app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (50, 50, [8, 7]),
+        (-15, 14.5, [0, 0]),
```

Comment:
The test case `(-15, 14.5, [0, 0])` includes negative and non-integer values for cat and dog ages. Ensure that the `get_human_age` function handles such edge cases appropriately, possibly by raising exceptions or returning a specific result as per the task requirements.